app_dart/lib/src/model/ci_yaml/target.dart

```diff
@@ -33,7 +33,7 @@ class Target {
   final RepositorySlug slug;
 
   /// Target prefixes that indicate it will run on an ios device.
-  static const List<String> iosPlatforms = <String>['mac_ios', 'mac_ios32'];
+  static const List<String> iosPlatforms = <String>['mac_ios', 'mac_arm64_ios'];
```

Comment:
Skipped the test as this is covered in https://github.com/flutter/cocoon/blob/main/app_dart/test/model/ci_yaml/target_test.dart#L46.